easybuild/tools/filetools.py

```diff
@@ -296,7 +408,7 @@ def find_easyconfigs(path, ignore_dirs=None):
                 continue
 
             spec = os.path.join(dirpath, f)
-            _log.debug("Found easyconfig %s" % spec)
+            _log.debug("Found easyconfig %s", spec)
```

Comment:
well, we should use `,`, if it works, that is if the log messages actually get formatted... I don't think that's the case (or at least I've run into unformatted log messages and/or error messages because of this)